app/models/podcast.rb

```diff
@@ -2,6 +2,7 @@ class Podcast < ApplicationRecord
   resourcify
 
   has_many :podcast_episodes
+  has_many :users, through: :podcast_ownerships
```

Comment:
is the goal to have multiple owners on a podcast? It might be beneficial to call this `.owners` instead of `.users`, similar to how we call `.creator` instead of `.user` the creator of the podcast

Reply:
see my note above about explicitly naming this `.owners`